tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -240,26 +240,26 @@ def query_db(
     return rows
 
 
-def get_metadata_sql(key_path: str) -> text:
+def get_metadata_sql() -> text:
     """
     Query for finding metadata based on key_path and PENDING status.
 
     Args:
-        key_path (str): s3 key for the file less the bucket name
+        None
     Returns:
         (sqlalchemy.text): SQL statement
     """
-    return text(  # nosec
-        f"""
+    return text(
+        """
             SELECT
                 job_id, granule_id, filename, restore_destination, multipart_chunksize_mb
             FROM
                 recovery_file
             WHERE
-                key_path = '{key_path}'
+                key_path = :key_path
             AND
-                status_id = {shared_recovery.OrcaStatus.PENDING.value}
-        """  # nosec
+                status_id = shared_recovery.OrcaStatus.PENDING.value
```

Comment:
Does this work? Seems like it would equeate to `shared_recovery.OrcaStatus.PENDING.value` instead of `0` or whatever the `value` is.

Reply:
it does return the rows as seen in unit tests

Reply:
Yup. Remember to update the docs, since it won't be specific to `PENDING` entries anymore.

Reply:
Updated. Let me know if I missed anywhere.